CHANGELOG.md

```diff
@@ -6091,3 +6091,4 @@
 [@nickpellant]: https://github.com/nickpellant
 [@friendlyantz]: https://github.com/friendlyantz
 [@issyl0]: https://github.com/issyl0
+[@ydah]: https://github.com/ydah
```

Comment:
This line is not needed as it is auto-generated by release task. Can you remove it?

Reply:
Sorry. I've deleted that line.